src/main.rs

```diff
@@ -102,8 +103,10 @@ fn build_response_json(people_present: Option<u32>, raspi_temperature: Option<f3
 }
 
 fn status_endpoint(_: Request, res: Response<Fresh>) {
-    let mut res = res.start().unwrap();
+    // Move response into mutable variable
+    let mut res = res;
```

Comment:
Why not declare it directly as mutable?
``fn status_endpoint(_: Request, mut res: Response<Fresh>)``